assets/js/components/feedback-prompt/index.js

```diff
@@ -13,7 +13,10 @@ import './style.scss';
 /**
  * Component to render a Feedback prompt in the sidebar.
  */
-const FeedbackPrompt = ( { text } ) => {
+const FeedbackPrompt = ( {
+	text,
+	url = 'https://ideas.woocommerce.com/forums/133476-woocommerce?category_id=384565',
+} ) => {
```

Comment:
Good call here with setting the default as the ideas board but allowing for overriding 👍